ckanext/xroad_integration/cli.py

```diff
@@ -66,6 +66,20 @@ def fetch_stats(ctx, config, days):
     with flask_app.test_request_context():
         utils.fetch_stats(days)
 
+
+@xroad.command(
+    u'fetch_distinct_service_stats',
```

Comment:
We're transistioning to click's preferred `kebab-case` names and click will figure out the correct command name automatically from the function name. This can be removed.

Reply:
the rest of the commands are fixed in my PR

Reply:
Done